src/components/GitStars.tsx

```diff
@@ -31,39 +30,18 @@ const GitStars = ({ gitHubRepo, hideStars, ...props }: GitStarsProps) => {
       textDecoration="none"
       {...props}
     >
-      <Flex
-        background="lightBorder"
-        textDecoration="none"
-        border="1px solid"
-        borderColor="lightBorder"
-        borderRadius="base"
-        color="text"
-        _hover={{
-          boxShadow: "0 0 1px var(--eth-colors-primary-base)",
-          path: { fill: "primary.base" },
-        }}
-      >
+      <Flex className="rounded border border-border bg-background-medium no-underline hover:fill-primary hover:shadow-table-item-box-hover">
```

Comment:
removing as it is redundant, border will default to the `border` color.
```suggestion
      <Flex className="rounded border bg-background-medium no-underline hover:fill-primary hover:shadow-table-item-box-hover">
```